trac-env/templates/site_head.html

```diff
@@ -1,3 +1,4 @@
+<meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
This was the line I thought I could merge, not the rest of the CSS (which I think we can improve). Do you think it makes sense to merge this `<meta>` tag on its own, or does it only make sense in conjunction with the other CSS changes?

Reply:
It is possible to merge without the CSS, but I think the CSS changes improved the mobile experience quite a bit - especially the scrolling.
Which parts of the CSS do you want to improve?

Reply:
Pulled out: https://github.com/django/code.djangoproject.com/pull/214